packages/block-library/src/navigation/theme.scss

```diff
@@ -3,4 +3,45 @@
 	ul li {
 		list-style: none;
 	}
+
+	ul > li > ul {
+		background: #e3e3e3;
```

Comment:
I think this is good. But if the goal is to make it stand out from the background, it won't work if the background is also gray :)

So we can either embrace that it might blend in, in which case I'd personally love a sliiiightly lighter gray. 

Or we can do something like add a shadow, gasp:

<img width="543" alt="Screenshot 2019-11-20 at 08 52 08" src="https://user-images.githubusercontent.com/1204802/69219473-349bed00-0b73-11ea-880e-3d1ef44bd7b0.png">

Up to you!